Makefile

```diff
@@ -11,8 +11,8 @@ VERSION ?= $(shell cat hokusai/VERSION)
 MINOR_VERSION ?= $(shell cat hokusai/VERSION | awk -F"." '{ print $$1"."$$2 }')
 
 dependencies:
-	pip install --requirement requirements.txt --quiet
-	pip install --quiet pyinstaller==3.3.1
```

Comment:
I added this as a dev dependency. See below.

https://github.com/artsy/hokusai/blob/b4f258f7ecd777c2a859f744ad7c81173dc21d5b/Pipfile#L7

Reply:
This needs to be installed on CI as part of the build toolchain - not sure if it makes sense here - see this failure https://circleci.com/gh/artsy/hokusai/675